app/src/main/res/values/styles.xml

```diff
@@ -55,23 +57,28 @@
 
     <style name="TextTitleSection" parent="Text">
         <item name="fontFamily">sans-serif</item>
-        <item name="android:lineSpacingExtra">6sp</item>
+        <item name="android:lineSpacingExtra">@dimen/default_line_spacing</item>
         <item name="android:textColor">@color/gnosis_dark_blue</item>
         <item name="android:textSize">16sp</item>
         <item name="android:textStyle">bold</item>
     </style>
 
     <style name="TextDark" parent="Text">
         <item name="fontFamily">sans-serif-medium</item>
-        <item name="android:lineSpacingExtra">6sp</item>
+        <item name="android:lineSpacingExtra">@dimen/default_line_spacing</item>
         <item name="android:textColor">@color/gnosis_dark_blue</item>
         <item name="android:textSize">16sp</item>
         <item name="android:textStyle">normal</item>
     </style>
 
+    <style name="TextDark.Bold" parent="TextDark">
+        <item name="fontFamily">sans-serif</item>
```

Comment:
This should be moved as well to where it's used.  I am not sure we are getting much value for defining a style that only sets 2 variables.

Reply:
If we can apply this style in more than one place, it makes sense to have it. Otherwise we will have to change this style in several places. 
This style is used (for example) also in the name of the safe in safe settings.